src/penn_chime/charts.py

```diff
@@ -74,7 +74,7 @@ def admitted_patients_chart(
 
     return (
         alt.Chart(census.head(plot_projection_days))
-        .transform_fold(fold=["Hospitalized Census", "ICU Census", "Ventilated Census"])
+        .transform_fold(fold=["Hospitalized", "ICU", "Ventilated"])
```

Comment:
This is just to get the graphs' x-axes to line up vertically. Label redundant to plot title and y-label.